core/src/main/java/ai/timefold/solver/core/impl/score/stream/bavet/common/BavetConcatConstraintStream.java

```diff
@@ -0,0 +1,8 @@
+package ai.timefold.solver.core.impl.score.stream.bavet.common;
```

Comment:
I wonder if these stream classes should be part of the package ` ai.timefold.solver.core.impl.bavet.common`

Reply:
I'm not making it too generic - when we need concat in data streams, we'll do it. I doubt we will need it there.